src/devtools/store.js

```diff
@@ -498,20 +498,26 @@ export default class Store extends EventEmitter {
     return list;
   }
 
-  getRendererIDForElement(id: number): number | null {
+  getRendererIDForElement(id: number | null): number | null {
```

Comment:
I don't think I agree with this type change. It makes no sense to call `getRendererIDForElement` with a null value. There could be no renderer ID for a non-element.